Configure.pl

```diff
@@ -164,12 +165,30 @@
     system($defaults{make}, 'realclean')
 }
 
+# test whether pkg-config works
+system("$config{pkgconfig}", "--version");
```

Comment:
Testing it this way spews out a message on systems that don't have it (for example, Windows).